gcloud-java-resourcemanager/src/main/java/com/google/gcloud/resourcemanager/ResourceManager.java

```diff
@@ -263,4 +291,72 @@ public static ProjectListOption fields(ProjectField... fields) {
+   * does not exist or if you do not have adequate permission to view the project.
+   *
+   * @see <a
+   * href="https://cloud.google.com/resource-manager/reference/rest/v1beta1/projects/getIamPolicy">
+   * Resource Manager getIamPolicy</a>
+   * @throws ResourceManagerException upon failure
+   */
+  Policy getPolicy(String projectId);
+
+  /**
+   * Sets the IAM access control policy for the specified project. Replaces any existing policy. The
+   * following constraints apply:
+   * <ul>
+   * <li>Projects currently support only <I>user:{emailid}</I> and <I>serviceAccount:{emailid}</I>
+   *     members in a binding of a policy.
+   * <li>To be added as an owner, a user must be invited via Cloud Platform console and must accept
+   *     the invitation.
+   * <li>Members cannot be added to more than one role in the same policy.
+   * <li>There must be at least one owner who has accepted the Terms of Service (ToS) agreement in
+   *     the policy. An attempt to set a policy that removes the last ToS-accepted owner from the
+   *     policy will fail.
+   * <li>Calling this method requires enabling the App Engine Admin API.
+   * </ul>
+   * Note: Removing service accounts from policies or changing their roles can render services
+   * completely inoperable. It is important to understand how the service account is being used
+   * before removing or updating its roles.
+   *
+   * It is recommended that you use the read-modify-write pattern. This pattern entails reading the
+   * project's current policy, updating it locally, and then sending the modified policy for
+   * writing. Cloud IAM solves the problem of conflicting processes simultaneously attempting to
+   * modify a policy by using the etag property. This property is used to verify whether the
+   * policy has changed since the last request. When you make a request to Cloud IAM with an etag
+   * value, Cloud IAM compares the etag value in the request with the existing etag value associated
+   * with the policy. It writes the policy only if the etag values match. If an etag is not
+   * provided, the policy is overwritten blindly.
+   *
+   * An example of using the read-write-modify pattern is as follows:
+   * <pre> {@code
+   * Policy currentPolicy = resourceManager.getPolicy("my-project-id");
+   * Policy modifiedPolicy =
+   *     current.toBuilder().removeIdentity(Role.VIEWER, Identity.user("user@gmail.com"));
+   * Policy newPolicy = resourceManager.setPolicy("my-project-id", modified);
+   * }
+   * </pre>
+   *
+   * @see <a href=
+   * "https://cloud.google.com/resource-manager/reference/rest/v1beta1/projects/setIamPolicy">
+   * Resource Manager setIamPolicy</a>
+   * @throws ResourceManagerException upon failure
+   */
+  Policy replacePolicy(String projectId, Policy newPolicy);
+
+  /**
+   * Returns the permissions that a caller has on the specified project. You typically don't call
```

Comment:
It might be worth clarifying how the input list of permissions relates to the output list of booleans. Also do you think a vararg overload of this method could be useful?